scripts/experiments/score_first_person_voice.py

```
@@ -0,0 +1,545 @@
+def iter_memory_docs(db: str) -> Iterable[dict[str, Any]]:
+    """Yield one dict per memory-carrying document in the DB.
+
+    Output shape:
+        {"_id": "<collection/_key>", "collection": "...", "field": "...",
+         "text": "..."}
+    """
+    for collection in MEMORY_COLLECTIONS:
+        try:
+            docs = _cursor(
+                db,
+                f"FOR d IN @@c LIMIT 100000 RETURN d",
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)